lab-doug/test/matchscore-router-test.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+const expect = require('chai').expect;
+const request = require('superagent');
+const server = require('../server');
+const storage = require('../lib/storage');
+const Matchscore = require('../model/matchscore');
+const port = process.env.PORT || 3000;
+const baseUrl = `localhost:${port}/api/matchscore`;
+
+
+
+describe('Testing matchscore-router module, ', function(){
+  before((done) => {
+    if(!server.isRunning){
+      server.listen(port, () => {
+        server.isRunning = true;
+        console.log('server is running on port: ', port);
+        done();
+      });
+      return;
+    }
+    done();
+  });
+
+  after((done) => {
+    if(server.isRunning){
+      server.close(() => {
+        console.log('server has been shutdown');
+        done();
+      });
+      return;
+    }
+    done();
+  });
+
+  describe('Testing POST on /api/matchscore endpoint', function(){
+    after((done) => {
```

Comment:
A lot of this seems like repeated code--in each case your after block is basically the same. That's a great time to use an afterEach, which does this same thing for you.